src/api/component.js

```diff
@@ -179,6 +179,7 @@ export default class extends HTMLElement {
     // DEPRECATED
     //
     // static render()
+    // Note that renderCallback is an optional method!
```

Comment:
It wasn't obvious to me. 
When we all worked on updating index.d.ts we didn't mark this method optional.
We now need to correct index.d.ts:
renderCallback?()